Mono.Cecil.Cil/Symbols.cs

```diff
@@ -408,7 +429,14 @@ public sealed class ImportDebugInformation : DebugInformation {
 		}
 
 		public Collection<ImportTarget> Targets {
-			get { return targets ?? (targets = new Collection<ImportTarget> ()); }
+			get
+			{
+				if (targets != null)
```

Comment:
Nit, is there a reason this pattern is inverted from the others?

Reply:
Nope, updated